components/Common/Pagination/index.module.css

```diff
@@ -0,0 +1,65 @@
+.pagination {
+  @apply grid
+    items-center
+    justify-between
+    gap-y-5
+    [grid-template-areas:'page-buttons_page-buttons_page-buttons''previous-button_._next-button']
```

Comment:
Interesting, I wonder why the choice of these complex area names... Couldn't you simplify it?

Reply:
Just wanted to be as informative as possible. What do you think of `pages`, `prev` and `next` as the new `grid-area` names?

Reply:
Sounds good to me